.github/workflows/build_and_test_debug_client.yml

```diff
@@ -187,7 +187,7 @@ jobs:
 
   ios_debug_build:
     name: iOS Debug Build
-    runs-on: macos-13
+    runs-on: macos-14
```

Comment:
This is because `VpnExtensionTest` requested `platform:iOS Simulator, OS:17.0.1`, which is [not supported on `macos-15`](https://github.com/Jigsaw-Code/outline-apps/actions/runs/20179118852/job/57934927897#step:8:131).